python/pyspark/mllib/linalg/__init__.py

```diff
@@ -846,6 +890,33 @@ def dense(*elements):
         return DenseVector(elements)
 
     @staticmethod
+    def fromML(vec):
+        """
+        Convert a vector from the new mllib-local representation.
+        This does NOT copy the data; it copies references.
+
+        >>> mllibDV1 = Vectors.dense([1, 2, 3])
+        >>> mlDV = newlinalg.Vectors.dense([1, 2, 3])
+        >>> mllibDV2 = Vectors.fromML(mlDV)
+        >>> mllibDV1 == mllibDV2
+        True
+        >>> mllibSV1 = Vectors.sparse(4, {1: 1.0, 3: 5.5})
+        >>> mlSV = newlinalg.Vectors.sparse(4, {1: 1.0, 3: 5.5})
+        >>> mllibSV2 = Vectors.fromML(mlSV)
+        >>> mllibSV1 == mllibSV2
+        True
+
+        :param vec: a :py:class:`pyspark.ml.linalg.Vector`
+        :return: a :py:class:`pyspark.mllib.linalg.Vector`
+        """
+        if type(vec) == newlinalg.DenseVector:
```

Comment:
It's a common pythonic practise to use `isinstance` in such cases. If we inherit something from `DenseVector`, then this check will fail.